jsl-tinylog/src/main/java/org/tinylog/jsl/TinylogLoggerFinder.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 Gerrit Rode
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package org.tinylog.jsl;
+
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+/**
+ * Logger finder for Java System.Logger that uses tinylog as back-end.
+ */
+public class TinylogLoggerFinder extends System.LoggerFinder {
+
+	private final ConcurrentMap<String, TinylogLogger> loggers = new ConcurrentHashMap<String, TinylogLogger>();
+
+	public TinylogLoggerFinder() {
+		super();
+	}
+
+	@Override
+	public System.Logger getLogger(final String name, final Module module) {
+		// How to handle modules as required by getLogger()?
+		// Should we check if there is an existing logger instance for name and module instead of just checking the name?
+
+		TinylogLogger logger = loggers.get(name);
```

Comment:
As an alternative this could be done via `loggers.computeIfAbsent( ... )`.  Is there a preference to do it this way over using `computeIfAbsent()` ?

Reply:
I would also like to use computeIfAbsent(...) but it does not work for Java 1.6. See the failed release build: `/home/runner/work/tinylog/tinylog/jsl-tinylog/src/main/java/org/tinylog/jsl/TinylogLoggerFinder.java:[32,69] method references are not supported in -source 1.6`

Reply:
Oh yes, jdk 6.  Quite right.

Reply:
Just a thought, this `jsl-tinylog` module depends on `System.Logger` and therefore has an implied dependency on JDK 9+ .  It seems that for this `jsl-tinylog` module we could use:
```
<maven.compiler.source>9</maven.compiler.source>
<maven.compiler.target>9</maven.compiler.target>
```
... which would enable the use of `computeIfAbsent()`?

Reply:
I fully agree! Feel free to change it if you want :)